### PR TITLE
Adds a script to collect all release notes since the last release

### PR DESCRIPTION
**Pull request by mohammedDehairy**
Adds a github action to collect all release notes since the last release

### CODE REVIEW

**Review by erenbesel — APPROVED** [review]

File: .github/workflows/get_release_notes.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    steps:
+      - name: Fetch repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get the list of allowed pull request labels
+        run: |
+          RED='\033[0;31m'
+          NC='\033[0m'
+          FILE_NAME=.pull-requests-allowed-labels-list
```
Comment (descorp) [inline .github/workflows/get_release_notes.yml:15]: How about we move this file to `.github/` folder ?

**Review by neelSharma12 — APPROVED** [review]

File: .github/workflows/get_release_notes.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          LATEST_TAG=$(git describe --tags --abbrev=0)
+          NUMBER_OF_COMMITS_SINCE_LAST_RELEASE=$(git log --oneline $LATEST_TAG..HEAD | wc -l | sed 's/^ *//g' | sed 's/ *$//g')
+
+          IFS=',' read -r -a ALLOWED_LABELS_ARRAY <<< "$ALLOWED_LABELS"
+
+          OUTPUT=""
+
+          for LABEL in "${ALLOWED_LABELS_ARRAY[@]}"; do
+             LABEL=$(echo "$LABEL" | sed 's/^ *//g' | sed 's/ *$//g')
+             GIT_LOG_CMD="git log -$NUMBER_OF_COMMITS_SINCE_LAST_RELEASE"
```
Comment (descorp) [inline .github/workflows/get_release_notes.yml:36]: This is `git log`, how does it get description from GitHub PRs? 🤔

Comment (mohammedDehairy) [reply .github/workflows/get_release_notes.yml:36]: the PR body is added to the merge commit message when merging the PR automatically, we just need to remember not to modify it when merging

File: .github/workflows/get_release_notes.yml (hunk 1 above)
Comment (mohammedDehairy) [reply .github/workflows/get_release_notes.yml:15]: this is in `.github/` folder already :-), this is a manual github work flow

Comment (descorp) [reply .github/workflows/get_release_notes.yml:15]: I mean `.pull-requests-allowed-labels-list` file

Comment (descorp) [reply .github/workflows/get_release_notes.yml:15]: It have nothing to do with SDK itself, so I see no point in keeping it in root 🤔

**Review by descorp — APPROVED** [review]

**Pull request merged**